### PR TITLE
Cl/facet fixes

### PR DESCRIPTION
**Pull request by CathLass**
request facets and pass through to use-case response

### CODE REVIEW

File: Dfe.Data.SearchPrototype/Common/Dfe.Data.SearchPrototype.Common.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
 
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (CathLass) [inline Dfe.Data.SearchPrototype/Common/Dfe.Data.SearchPrototype.Common.csproj]: remove

File: Dfe.Data.SearchPrototype/Dfe.Data.SearchPrototype.csproj [hunk 2]
```diff
@@ -4,7 +4,9 @@
 		<TargetFramework>net8.0</TargetFramework>
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
-
+		
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (CathLass) [inline Dfe.Data.SearchPrototype/Dfe.Data.SearchPrototype.csproj]: remove

File: Dfe.Data.SearchPrototype/Infrastructure/Dfe.Data.SearchPrototype.Infrastructure.csproj [hunk 3]
```diff
@@ -5,6 +5,8 @@
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
 
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (CathLass) [inline Dfe.Data.SearchPrototype/Infrastructure/Dfe.Data.SearchPrototype.Infrastructure.csproj]: remove

**Review by CathLass — COMMENTED** [review]
some xml comment warnings
are we testing the new functionality added to the use-case and the adaptor? (I've run out of concentration)

**Review by AsiaWi — APPROVED** [review]

**Pull request merged**